modules/apps/27-interchain-accounts/host/client/cli/tx.go

```diff
@@ -0,0 +1,145 @@
+package cli
+
+import (
+	"encoding/json"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	icatypes "github.com/cosmos/ibc-go/v6/modules/apps/27-interchain-accounts/types"
+)
+
+const (
+	memoFlag string = "memo"
+)
+
+func generatePacketDataCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "generate-packet-data [message]",
+		Short: "Generates ICA packet data.",
+		Long: `generate-packet-data accepts a message string and serializes it
+into packet data which is outputted to stdout. It can be used in conjunction with send-tx"
+which submits pre-built packet data containing messages to be executed on the host chain.
+`,
+		Example: `<binary> tx interchain-accounts host generate-packet-data '{
```

Comment:
Can we use `version.AppName` instead of `<binary>` like in [here](https://github.com/cosmos/ibc-go/blob/main/modules/apps/27-interchain-accounts/host/client/cli/query.go#L28)?